android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/service/BdsService.java

```diff
@@ -111,6 +111,10 @@ public void makeRequest(String endPoint, String httpMethod, List<String> paths,
     AsyncTask asyncTask =
         new ServiceAsyncTask(this, baseUrl, endPoint, httpMethod, paths, queries, body,
             serviceResponseListener);
-    asyncTask.execute();
+    if (async) {
```

Comment:
why did you add async flag?
is there any usage of it as false?

Reply:
As of now this variable will never be false, but since this is suppose to be a generic class, I'll leave here anyway. Changed the name tho, as of now it does not portrait what it intended (parallelism)